packages/client/src/components/Question/Question.tsx

```diff
@@ -105,7 +106,7 @@ const Question: FunctionComponent<QuestionProps> = ({
         </ComponentWrapper>
       )}
 
-      <Answers {...{ errors, question, saveAnswer }} />
+      <Answers {...{ collection, errors, question, saveAnswer }} />
```

Comment:
No need to pass `collection` because the entire `question` is passed.

Reply:
already fixed.